rpc/handler.go

```diff
@@ -388,7 +406,23 @@ func (h *handler) handleCallMsg(ctx *callProc, msg *jsonrpcMessage, stream *json
 		}
 		return nil
 	case msg.isCall():
+		if h.rpcSlowLogThreshold > 0 && h.isRpcMethodNeedsCheck(msg.Method) {
+			slowTimer := time.AfterFunc(time.Duration(h.rpcSlowLogThreshold)*time.Millisecond, func() {
```

Comment:
really interesting. looks good for me

let's calc `time.Duration(h.rpcSlowLogThreshold)*time.Millisecond` once - when parsing cli flag.